src/System.Reflection.Metadata/specs/Ecma-335-Issues.md

```diff
@@ -0,0 +1,133 @@
+# Known Issues in ECMA-335 CLI Specification 
```

Comment:
Right now, I just wanted to save my notes. I'm not even sure what we should consider a spec bug and what we should consider implementation issues yet. I'd love to continue the conversation about the next steps towards where these notes should live, how they should be split, and the path towards amending the spec where appropriate, etc,, but I'd rather not block this PR on it.